### PR TITLE
Updated tool chain for converting between substrate account and evm address

### PR DESCRIPTION
**Pull request by CaptainVincent**
Update tool chain enhance airdrop usability.

### CODE REVIEW

File: template/utils/package.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "name": "substrate-evm-utils",
   "version": "0.0.0",
   "dependencies": {
-    "@polkadot/util-crypto": "^2.2.1",
+    "@polkadot/util-crypto": "^6.6.1",
+    "@polkadot/uti": "^6.6.1",
```
Comment (dm4) [inline template/utils/package.json]: missing `l`?

**Comment by dm4** [comment]
Should we also update `README.md`?

File: template/utils/evm-account.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+module.exports = () => {
+  if (process.argv.length < 4) {
+    console.error('Please provide the <account> or <address> parameter.');
+    console.error(help);
+    process.exit(9);
+  }
+
+  const address = process.argv[3];
+  if (!address.match(/^[A-z0-9]{40}|0x[A-z0-9]{40}|[A-z0-9]{48}$/)) {
```
Comment (dm4) [inline template/utils/evm-account.js]: I think using `[A-Fa-f0-9]{40}` for evm addresses is better here?

Comment (CaptainVincent) [reply template/utils/evm-account.js]: good suggestion

File: template/utils/package.json (hunk 1 above)
Comment (CaptainVincent) [reply template/utils/package.json]: yes.

**Comment by CaptainVincent** [comment]
> Should we also update `README.md`?

Currently we don't have any customize modify in README.md. 
I think maybe whole document update could be a separate task in the future.

**Review by dm4 — APPROVED** [review]
LGTM

**Pull request merged**